### PR TITLE
[tools] config tool fullnode-init subcommand

### PR DESCRIPTION
**Pull request by sirouk**
- [x] `libra config fullnode-init` sets up the basic files for starting a fullnode
- [x] downloads a genesis.blob and waypoint.txt
- [x] generates a yaml and updates seed peers from a community list

### CODE REVIEW

**Review by codilion — APPROVED** [review]
🤘

File: tools/config/src/fullnode_config.rs [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  enabled: true
+  address: '0.0.0.0:8080'
+"
+    );
+    Ok(template)
+}
+
+/// download genesis blob
+pub async fn download_genesis(home_dir: Option<PathBuf>) -> anyhow::Result<()> {
+    let bytes = reqwest::get("https://github.com/0xzoz/blob/raw/main/genesis.blob")
```
Comment (0xzoz) [inline tools/config/src/fullnode_config.rs:94]: Should we have repositories stored in the 0LNetworkCommunity repo for this? Seperate repos for REX and Mainnet

**Review by 0xzoz — APPROVED** [review]

**Review by hemulin — APPROVED** [review]
LGTM

File: tools/config/src/fullnode_config.rs (hunk 1 above)
Comment (0o-de-lally) [reply tools/config/src/fullnode_config.rs:94]: yeah I have this as a placeholder. 
Perhaps keep it in the original network genesis repo, under a directory: `upgrades`

**Pull request merged**